src/main/kotlin/de/docs_as_co/intellij/plugin/drawio/editor/BaseDrawioWebView.kt

```diff
@@ -105,6 +103,13 @@ abstract class BaseDrawioWebView(val lifetime: Lifetime) {
 
     private var requestId = 0
 
+    public fun reload(uiTheme: String) {
+        this.uiTheme = uiTheme
+        //initializeSchemeHandler(uiTheme)
+        this.panel.browser.cefBrowser.reload()
+        send(OutgoingMessage.Event.Load("",1))
```

Comment:
What is `Load` doing?

Reply:
this is part of the WIP - doesn't work yet. 
The `reload()` results in having only the menu. I think there is a load event missing at this point.

Reply:
removed it and replaced it with a todo comment